docs/API.md

```diff
@@ -107,17 +107,23 @@ A `<Link>` can know when the route it links to is active and automatically apply
 
 #### Props
 ##### `to`
-The path to link to, e.g. `/users/123`.
-
-##### `query`
+Can be either a string or an object.
```

Comment:
Your description below is accurate in most cases, but not all cases. The value of `to` is passed to the history as-is. Usually it will behave as you note, but a custom history can do whatever.